MIDIchlorians/MIDIchlorians/Config.swift

```diff
@@ -39,6 +39,7 @@ struct Config {
             "AWOLNATION - Sail-28", "AWOLNATION - Sail"
         ]
     ]
+    static let SoundExt = "wav"
```

Comment:
Can also, in that case the file name saved in the database will include ".wav" at the back, that's fine?